FlowCrypt/src/test/java/com/flowcrypt/email/api/email/WkdClientTest.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * © 2021-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors:
+ *   Ivan Pizhenko
+ */
+
+package com.flowcrypt.email.api.email
+
+import com.flowcrypt.email.api.wkd.WkdClient
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class WkdClientTest {
+  @Test
+  fun existingEmailTest() {
+    val keys = WkdClient.lookupEmail("human@flowcrypt.com")
+    assertTrue("Key not found", keys != null)
+    assertTrue("There are no keys in the key collection", keys!!.keyRings.hasNext())
+  }
+
+  @Test
+  fun nonExistingEmailTest() {
+    val keys = WkdClient.lookupEmail("no.such.email.for.sure@flowcrypt.com")
+    assertTrue("Key found for non-existing email", keys == null)
+  }
```

Comment:
Looks great, should also test the following:
 - doesnotexist@thisdomaindoesnotexist.test
 - doesnotexist@google.com

In both cases, it should return no pubkey without throwing or failing.

Timeout is 3 seconds when it cannot reach, then it returns 'no pubkey'.

Reply:
@tomholub 
I've added tests with these emails.
In TypeScript timeout was 4 seconds, so I've used that.
Not sure how to test what will happen on timeout expiration.